Exiled.API/Features/Map.cs

```diff
@@ -127,6 +129,32 @@ public static void Broadcast(ushort duration, string message, global::Broadcast.
             Server.Broadcast.RpcAddElement(message, duration, type);
         }
 
+        /// <summary>
+        /// Shows a Hint to all players.
+        /// </summary>
+        /// <param name="duration">The duration in seconds.</param>
+        /// <param name="message">The message that will be broadcast (supports Unity Rich Text formatting).</param>
+        public static void ShowHint(float duration, string message)
+        {
+            foreach (Player player in Player.List)
+            {
+                player.ShowHint(message, duration);
+            }
+        }
```

Comment:
Two methods, why? I prefer `ShowHint(string, float)` as in the player class.

Reply:
Was trying to go for more flexibility, I was thinking of maybe requesting a change for ShowHint on the Player class, since there's not much of a reason for ShowHint to have a different Syntax from the Broadcast method, but that may be better to be discussed on the Discord chat. :p

Reply:
I really don't like implementing a method with a different order of the same arguments. This may be confusing for a developer as you can see the issue https://github.com/galaxy119/EXILED/issues/112